### PR TITLE
More best practices fixes

### PR DESCRIPTION
**Pull request by ricardoquesada**
- Removes Hungarian notation from the CPP test (there are still some Hungarian to remove)
- Label and MotionStreak: Uses the CL() entry for the tests (lambda functions)

### CODE REVIEW

File: samples/Cpp/TestCpp/Classes/PerformanceTest/PerformanceSpriteTest.cpp [hunk 1]
```diff
@@ -408,77 +408,77 @@ void SpriteMainScene::onDecrease(Object* pSender)
 // For test functions
 //
 ////////////////////////////////////////////////////////
-void performanceActions(Sprite* pSprite)
+void performanceActions(Sprite* scene)
```
Comment (dumganhar) [inline samples/Cpp/TestCpp/Classes/PerformanceTest/PerformanceSpriteTest.cpp]: `pSprite` --> `scene` ?

Comment (dumganhar) [inline samples/Cpp/TestCpp/Classes/PerformanceTest/PerformanceSpriteTest.cpp]: Lots of `pSprite` --> `scene`, wrong replacement?

Comment (ricardoquesada) [inline samples/Cpp/TestCpp/Classes/PerformanceTest/PerformanceSpriteTest.cpp]: ouch. thanks. I'll fix it.

**Comment by ricardoquesada** [comment]
@dumganhar fixed, and I added other fixes.

As you can see, it won't compile, but this is because the JS bindings are outdated.
I renamed a few methods (that were renamed recently, but the new names weren't C++ friendly), and that's why the JS bindings and Lua bindings are not compiling.

thanks.

**Comment by dumganhar** [comment]
> As you can see, it won't compile, but this is because the JS bindings are outdated.
> I renamed a few methods (that were renamed recently, but the new names weren't C++ friendly), and that's why the JS bindings and Lua bindings are not compiling.

OK, I'll check it. I like the new naming. :)
Thanks.

**Comment by minggo** [comment]
Hi @ricardoquesada 
There is a header file named `CCDeprecated.h` in include.
We will move `CCLog` into it.

**Comment by ricardoquesada** [comment]
Thanks @minggo.
Next time I will move the deprecated functions there.

**Pull request merged**